barretenberg/cpp/src/barretenberg/stdlib/hash/blake3s/blake3s.hpp

```diff
@@ -38,31 +36,31 @@ template <typename Builder> class Blake3s {
                                                  0x510E527FUL, 0x9B05688CUL, 0x1F83D9ABUL, 0x5BE0CD19UL };
 
     struct blake3_hasher {
-        field_t<Builder> key[8];
-        field_t<Builder> cv[8];
-        byte_array<Builder> buf;
+        field_ct key[8];
+        field_ct cv[8];
+        byte_array_ct buf;
         uint8_t buf_len;
         uint8_t blocks_compressed;
         uint8_t flags;
         Builder* context;
     };
 
     struct output_t {
-        field_t<Builder> input_cv[8];
-        byte_array<Builder> block;
+        field_ct input_cv[8];
+        byte_array_ct block;
         uint8_t block_len;
         uint8_t flags;
     };
-    static void compress_pre(field_t<Builder> state[BLAKE3_STATE_SIZE],
-                             const field_t<Builder> cv[8],
+    static void compress_pre(field_ct state[BLAKE3_STATE_SIZE],
+                             const field_ct cv[8],
                              const byte_array_ct& block,
                              uint8_t block_len,
                              uint8_t flags);
 
-    static void compress_in_place(field_t<Builder> cv[8], const byte_array_ct& block, uint8_t block_len, uint8_t flags);
+    static void compress_in_place(field_ct cv[8], const byte_array_ct& block, uint8_t block_len, uint8_t flags);
 
     static void compress_xof(
```

Comment:
for next PR: If any of these functions are unused (like unused as a black box function in noir), please remove it.